src/main/resources/application.yaml

```diff
@@ -331,8 +332,7 @@ darts:
         housekeeping: ecfd1f14-c9b6-4f15-94c7-cc60e53f2c7a
         dailylist-processor: f6f71122-ff85-4ebe-93d9-1706460dbea5
       unstructured-to-arm-batch-processor:
-        max-arm-manifest-items: 100
-        max-arm-single-mode-items: 100
+        max-result-size: 100000
```

Comment:
Why do we have the batch size hardcoded in yaml. It should be fetched from DB which the existing code is already doing.

Reply:
I think that was a merge conflict. Will correct it.